src/main/webapp/assets/modules/get-promoted/gp-form.mod.js

```diff
@@ -1,28 +1,34 @@
-techlooper
-  .directive("getPromotedForm", function ($http, vnwConfigService) {
-    return {
-      restrict: "E",
-      replace: true,
-      templateUrl: "modules/get-promoted/gp-form.tem.html",
-      link: function (scope, element, attr, ctrl) {
-        scope.doPromotion = function () {
-          if (!scope.promotionForm.$valid) {
-            return false;
-          }
-          scope.masterPromotion = angular.copy(scope.promotionInfo);
-          $http.post("getPromoted", scope.promotionInfo)
-            .success(function (data, status, headers, config) {
-              scope.masterPromotion.result = data;
-              scope.changeState('result');
-            });
-        }
+techlooper.directive("getPromotedForm", function ($http, vnwConfigService, $location) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/get-promoted/gp-form.tem.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.